data/org.freedesktop.impl.portal.Settings.xml

```diff
@@ -43,6 +43,29 @@
           Unknown values should be treated as 0 (no preference).
         </para></listitem>
       </varlistentry>
+      <varlistentry>
+        <term>org.freedesktop.appearance button-placement (asas)</term>
+        <listitem><para>
+          Indicates the system's preferred arrangement of buttons on the titlebar.
+          In LTR locales, the first tuple is on the left (leading), and the second tuple is on the right (trailing).
+          In RTL locales, the first tuple is on the right (leading), and the second tuple is on the left (trailing).
+          Supported values are:
+          <simplelist>
+            <member>close: the close button</member>
+            <member>minimize: the minimize button</member>
+            <member>maximize: the maximize button</member>
+          </simplelist>
+          Unknown values must be ignored if the client does
+          not understand them. Duplicate buttons across the tuples and
+          string arrays are not allowed.
```

Comment:
@ebassi @jadahl do you both have any thoughts on other paths this could take? anything nicer to expose over dbus? anything that's easier to put into code? it is Not Fun™ as-is.